### ISSUE TITLE
Overhaul user popover

### ISSUE THREAD

**Issue by BurntimeX** [WoltLab/WCF#3400]
- [ ] Make use of the cover photo
- [ ] Reduce the amount of shown information to prevent overflows

ref https://community.woltlab.com/thread/282419-inhalt-von-profilfeldern-wird-in-der-benutzerprofil-vorschau-u-u-abgeschnitten/

**Comment by eveneme** [comment]
Indeed, there is room for improvement. It would be better to rearrange the text and icons for greater clarity and readability.

Small sugestion: 
![image](https://github.com/WoltLab/WCF/assets/85762221/5b33d736-022f-4282-a72b-c066d904e23f)
